### ISSUE TITLE
ST6 4FA is missed

### ISSUE THREAD

**Issue by lboff** [ideal-postcodes/postcodes.io#950]
ST6 4FA is missed in https://api.postcodes.io but it is a valid post code

**Comment by jimbo1999uk** [comment]
That postcode has only just appeared in the February 2022 release of the ONSPD but the library hasn't been updated yet and still uses the November data, so the postcode won't be visible yet. Once the data has been updated it should be available.

**Comment by lboff** [comment]
Is there a time plan for this. Can I help in some way?

**Comment by cblanc** [comment]
#951 makes ONSPD Feb 22 is available 

It will be available on docker hub as 14.3.0 in about 10 minutes or so. Will likely deploy this to api.postcodes.io tomorrow

`ST6 4FA` appears in this release

```json
{
    "status": 200,
    "result": {
        "postcode": "ST6 4FA",
        "quality": 5,
        "eastings": 385778,
        "northings": 350400,
        "country": "England",
        "nhs_ha": "West Midlands",
        "longitude": -2.213613,
        "latitude": 53.050685,
        "european_electoral_region": "West Midlands",
        "primary_care_trust": "Stoke on Trent",
        "region": "West Midlands",
        "lsoa": "Stoke-on-Trent 005C",
        "msoa": "Stoke-on-Trent 005",
        "incode": "4FA",
        "outcode": "ST6",
        "parliamentary_constituency": "Stoke-on-Trent North",
        "admin_district": "Stoke-on-Trent",
        "parish": "Stoke-on-Trent, unparished area",
        "admin_county": null,
        "admin_ward": "Burslem Central",
        "ced": null,
        "ccg": "NHS Stoke on Trent",
        "nuts": "Stoke-on-Trent",
        "codes": {
            "admin_district": "E06000021",
            "admin_county": "E99999999",
            "admin_ward": "E05008723",
            "parish": "E43000017",
            "parliamentary_constituency": "E14000973",
            "ccg": "E38000175",
            "ccg_id": "05W",
            "ced": "E99999999",
            "nuts": "TLG23",
            "lsoa": "E01014356",
            "msoa": "E02002955",
            "lau2": "E06000021"
        }
    }
}
```

**Comment by cblanc** [comment]
Update has been deployed
